src/tensorial_form.jl

```diff
@@ -17,10 +17,16 @@ Compute the Bernstein expansion in tensorial form given its implicit form.
     * `"loop"`
     * `"kron"`
 """
-function generate_tensor_form(B::ImplicitForm{N}; algorithm::String="loop") where {N} 
+function generate_tensor_form(B::ImplicitForm{N}; algorithm::String="loop") where {N}
     # unpack fields
     v = VectorOfArray(B.array) # FIXME : move to type?
-    generate_tensor_form(v, algorithm=algorithm)
+    if algorithm == "naive"
+        A = ones(typeof(v[1,1]),Tuple(length(vi) for vi in v))
+        _naive!(A,B.ncoeffs,v)
```

Comment:
i would suggest to eliminate this function (merge it) in favor of the other one with the same name, ie.  such that the algorithm `naive` can be called from `generate_tensor_form(v::VectorOfArray{N, D, T}; algorithm::String="loop") where {N, D, T}`, right?

Reply:
In the `generate_tensor_form(v::VectorOfArray{N, D, T}; algorithm::String="loop") where {N, D, T}` function some arguments are lacking for the naive code.
If it were me I would just keep the function `generate_tensor_form(B::ImplicitForm....)`

Reply:
okay, can you merge all into the `generate_tensor_form` that receives an implicit form?